For some reason, it's not enough to source this file and then override the variable.

We have to patch it. It must be getting sourced again by internal Quarto code
at some point.

diff --git a/configuration b/configuration
index 0b6bef3..557ef9c 100644
--- a/configuration
+++ b/configuration
@@ -44,7 +44,7 @@ export ALGOLIA_SEARCH_INSIGHTS_JS=2.0.3
 
 
 # Quarto Info Version
-export QUARTO_VERSION=1.3
+export QUARTO_VERSION=${PKG_VERSION}
 export QUARTO_NAME=Quarto
 export QUARTO_BUILD_RUN_OFFSET=-1690
 
